contrib/camel/config/205-camel-k-integrationcontext.yaml

```diff
@@ -0,0 +1,44 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  labels:
+    app: camel-k
+  name: integrationcontexts.camel.apache.org
+spec:
+  additionalPrinterColumns:
+  - JSONPath: .status.phase
```

Comment:
nice. We should add custom columns into the other resources now that it is better supported in k8s.